tight/providers/aws/controllers/lambda_proxy_event.py

```
@@ -130,10 +130,8 @@ def run(self, *args, **kwargs):
             trace_lines = [re.sub(r'^(\s+)', (len(re.match(r'^(\s+)', line).groups(0)[0]) * '_') + ' ', line) if re.match(r'^\s+', line) is not None else line for line in trace_lines]
             trace = "\n".join(trace_lines)
             error(message='\n\n' + trace)
-            prepared_response = {
-                'statusCode': 500,
-                'body': 'There was an error.'
-            }
+            raise Exception('There was an error.')
```

Is this still doing anything with the three lines above it? Should it raise a more meaningful error or is that handled in the line above?

---

error is logging the stack trace - main reason for the catching of the exception. Instead of returning a 'hand crafted' 500 response, we're throwing generic exception so the x-ray runtime sees the 500 as a fault (previously x-ray the function finish OK)